src/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/memory_LSTMCell.hpp

```diff
@@ -5,43 +5,49 @@
 #pragma once
 
 #include "common_test_utils/test_common.hpp"
-#include "shared_test_classes/base/layer_test_utils.hpp"
-#include <ie_core.hpp>
-
-namespace SubgraphTestsDefinitions {
-typedef std::tuple<
-    ngraph::helpers::MemoryTransformation,   // Apply Memory transformation
-    std::string,                        // Target device name
-    InferenceEngine::Precision,         // Network precision
-    size_t,                             // Input size
-    size_t,                             // Hidden size
-    std::map<std::string, std::string>  // Configuration
-> memoryLSTMCellParams;
-
-class MemoryLSTMCellTest : virtual public LayerTestsUtils::LayerTestsCommon,
-    public testing::WithParamInterface<memoryLSTMCellParams> {
+#include "ov_models/utils/ov_helpers.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+typedef std::tuple<ov::test::utils::MemoryTransformation,  // Apply Memory transformation
+                   std::string,                            // Target device name
+                   ov::element::Type,                      // Input element type
+                   size_t,                                 // Input size
+                   size_t,                                 // Hidden size
+                   ov::AnyMap                              // Configuration
+                   >
+    memoryLSTMCellParams;
+
+class MemoryLSTMCellTest : virtual public ov::test::SubgraphBaseStaticTest,
+                           public testing::WithParamInterface<memoryLSTMCellParams> {
 private:
-    // you have to Unroll TI manually and remove memory untill ngraph supports it
+    // you have to Unroll TI manually and remove memory until ngraph supports it
```

Comment:
Detail: As is migrate the `ngraph` should be removed or replaced. Check others comments